src/shipit_signoff/tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+import pytest
+import os
+import shipit_signoff.flask  # NOQA
```

Comment:
the error you are getting comes from this line. I'm looking currently how to get testing infrastructure nicer.